src/Billing/Utilities/PayPalIpnClient.cs

```diff
@@ -30,6 +32,7 @@ public async Task<bool> VerifyIpnAsync(string ipnBody)
             Method = HttpMethod.Post,
             RequestUri = _ipnUri
         };
+        _httpClient.DefaultRequestHeaders.Add("User-Agent", "CSharp-IPN-VerificationScript");
         var cmdIpnBody = string.Concat("cmd=_notify-validate&", ipnBody);
```

Comment:
Could you also add an error log on line 25, before throwing the `ArgumentException`?
https://github.com/bitwarden/server/pull/3338/files#diff-39b4009fa5c0ead246f5ddccad5dd1afd681e6b7b9db9f157c9e9df99b9917d5L25